app/cli/cmd/root.go

```diff
@@ -174,6 +184,18 @@ func NewRootCmd(l zerolog.Logger) *cobra.Command {
 	return rootCmd
 }
 
+// this could have been done using automatic + prefix but we want to have control and know the values
```

Comment:
What's the issue otherwise?

Reply:
The issue is that it's not trivial to know what's the resulting env var name to be shown in the CLI. I have mixed feelings in any case, we could go with either.